Jenkinsfile

```diff
@@ -9,8 +9,15 @@ pipeline {
     timeout(time: 2, unit: 'HOURS')
     disableConcurrentBuilds()
   }
+
   stages {
 
+    stage('nvidia-smi'){
```

Comment:
Not sure if you were doing this just to see what GPUs were on the CI machine?

Reply:
Yup, just handy to have in the pipeline.